ivadomed/loader/utils.py

```diff
@@ -505,3 +507,78 @@ def reorient_image(arr, slice_axis, nib_ref, nib_ref_canonical):
     trans_orient = nib.orientations.ornt_transform(ras_orientation, ref_orientation)
     # apply transformation
     return nib.orientations.apply_orientation(arr_ras, trans_orient)
+
```

Comment:
PEP8 asks for 2 lines between function defs

Reply:
Fixed in 85537da